contracts/libraries/Commands.sol

```diff
@@ -38,37 +38,15 @@ library Commands {
     // The following constant defines one of the boundaries where the if blocks split commands
     uint256 constant SECOND_IF_BOUNDARY = 0x10;
 
-    // Command Types where 0x10<=value<0x18, executed in the third nested-if block
```

Comment:
should we make the command "enum" numbers make more sense?

Reply:
made an issue so we can revisit after v4 goes in